<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>About</class>
 <widget class="QDialog" name="About">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>About Tano</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../data/logo.qrc">
    <normaloff>:/logo/256x256/logo.png</normaloff>:/logo/256x256/logo.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="pageAbout">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1" colspan="3">
        <layout class="QGridLayout" name="mainGridLayout">
         <item row="0" column="1" colspan="4">
          <widget class="QLabel" name="labelName">
           <property name="pixmap">
            <pixmap resource="../../data/images.qrc">:/images/name.png</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2" rowspan="2">
          <widget class="QLabel" name="labelVersion">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="3" rowspan="2">
          <widget class="QLabel" name="labelChangeset"/>
         </item>
         <item row="1" column="1" rowspan="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="4" rowspan="2">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="1" colspan="4">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" rowspan="4">
          <widget class="QLabel" name="labelLogo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>128</width>
             <height>128</height>
            </size>
           </property>
           <property name="pixmap">
            <pixmap resource="../../data/logo.qrc">:/logo/128x128/logo.png</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="labelCopy">
         <property name="text">
          <string notr="true">&lt;h4&gt;&amp;copy; 2008-%1 Tadej Novak&lt;/h4&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="labelLicense">
         <property name="text">
          <string>&lt;h4&gt;Licensed under GNU GPL&lt;/h4&gt;</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="labelLinks">
         <property name="text">
          <string notr="true">&lt;h4&gt;http://tanoplayer.co.cc&lt;br&gt;info@tanoplayer.co.cc&lt;/h4&gt;</string>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="3">
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::Ok</set>
         </property>
         <property name="centerButtons">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="3" rowspan="3">
        <layout class="QGridLayout" name="libGridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="labelIconQt">
           <property name="maximumSize">
            <size>
             <width>28</width>
             <height>32</height>
            </size>
           </property>
           <property name="pixmap">
            <pixmap resource="../../data/icons.qrc">:/icons/other/qt.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" rowspan="2">
          <widget class="QLabel" name="labelIconVlcqt">
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="pixmap">
            <pixmap resource="../../data/icons.qrc">:/icons/other/vlcqt.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="labelQVlc"/>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="labelVlc">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="labelQt">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="2" rowspan="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageLicense">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="License" name="widgetLicense" native="true"/>
       </item>
       <item>
        <widget class="QPushButton" name="buttonLicense">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>License</class>
   <extends>QWidget</extends>
   <header>ui/License.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../data/icons.qrc"/>
  <include location="../../data/images.qrc"/>
  <include location="../../data/logo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>About</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
